src/integrationTest/java/uk/gov/hmcts/reform/notificationservice/config/SchedulerConfigurationTest.java

```diff
@@ -6,12 +6,17 @@
 import org.mockito.ArgumentCaptor;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.boot.test.mock.mockito.SpyBean;
+import uk.gov.hmcts.reform.notificationservice.task.PendingNotificationsTask;
 
-import static org.mockito.Mockito.never;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.atLeastOnce;
 import static org.mockito.Mockito.verify;
 
 @SpringBootTest(properties = {
-    "scheduling.task.notifications-consume.enable=true" // task is not there yet, but config is
+    // consume task is not under shedlock. including config for future in case it will be
+    "scheduling.task.notifications-consume.enable=true",
```

Comment:
It would nice to add it when needed else we might not realise what else is missing :)

Reply:
hence adding now so we realise that we missed it :). now it works as expected - win win